graphql_compiler/tests/test_compiler.py

```diff
@@ -4237,12 +4318,71 @@ def test_fold_same_edge_type_in_different_locations(self):
                   schema_1."Animal" AS "Animal_5"
                 JOIN schema_1."Animal" AS "Animal_6" ON "Animal_5".uuid = "Animal_6".parent
                 GROUP BY
-                  "Animal_5".uuid
+                    "Animal_5".uuid
             ) AS folded_subquery_2 ON "Animal_4".uuid = folded_subquery_2.uuid
         '''
+        expected_mssql = '''
+            SELECT
+              [Animal_1].name AS animal_name,
+              folded_subquery_1.fold_output_name AS child_names_list,
+              folded_subquery_2.fold_output_name AS sibling_and_self_names_list
+            FROM
+              db_1.schema_1.[Animal] AS [Animal_1]
+              LEFT OUTER JOIN(
+                SELECT
+                  [Animal_2].uuid AS uuid,
+                  coalesce((
+                    SELECT
+                      :coalesce_1 + coalesce(
+                        REPLACE(
+                          REPLACE(
+                            REPLACE([Animal_3].name, :REPLACE_1, :REPLACE_2),
+                            :REPLACE_3,
```

Comment:
Is this right? Do we expect these to be query parameters? I don't think they get added to the query parameter list during execution, I would have expected them to be literal characters instead.